src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -113,6 +115,11 @@ internal WebApplicationBuilder(WebApplicationOptions options, Action<IHostBuilde
     /// </summary>
     public ConfigureHostBuilder Host { get; }
 
+    /// <summary>
+    /// An <see cref="AuthenticationBuilder"/> for configuration authentication-related properties.
+    /// </summary>
+    public AuthenticationBuilder Authentication { get; }
```

Comment:
```suggestion
    public AuthenticationBuilder Authentication => _webAuthBuilder;
```

Then we should add a `private readonly WebApplicationAuthenticationBuilder _webAuthBuilder;` field so we don't have to cast later.

Reply:
Good call!